### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected a dependency for the `laravel/framework`. Unless you have a special requirement for depending on the entire Laravel framework, it's recommended to add dependencies for the specific `illuminate` packages instead.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laratrust.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "santigarcor/laratrust": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4, 7.3, 7.2, 8.0]
```
Comment (jasonmccreary) [inline .github/workflows/tests.yml:11]: You may preserve all these if you want to remain backwards compatible. Although I would recommend dropping PHP 7.2. 😉

File: composer.json [hunk 2]
```diff
@@ -23,11 +23,11 @@
     "require": {
         "php": ">=7.2",
         "kkszymanowski/traitor": "^0.2.0",
-        "laravel/framework": "~6.0|~7.0|~8.0"
+        "laravel/framework": "~6.0|~7.0|~8.0|^9.0"
     },
     "require-dev": {
         "mockery/mockery": "^1.3.2",
-        "orchestra/testbench": "4.*|5.*|6.*",
+        "orchestra/testbench": "4.*|5.*|6.*|dev-remove-nice-to-have-deps",
```
Comment (jasonmccreary) [inline composer.json:30]: This should be `7.*`. There was a mistag in this package the other day.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (KKSzymanowski) [reply .github/workflows/tests.yml:11]: Both PHP 7.2 and 7.3 are no longer maintained so we can probably drop the support for them if we're doing a new major version but if we want to go with a minor version for compatibility with L9 I would keep all of these PHP versions.

**Comment by SupianIDz** [comment]
`src/Console/AddLaratrustUserTraitUseCommand.php:69` should return `string` on return type hint.

Screen Shot 2022-01-30 at 12 28 54

File: composer.json [hunk 3]
```diff
@@ -23,11 +23,11 @@
     "require": {
         "php": ">=7.2",
         "kkszymanowski/traitor": "^0.2.0",
-        "laravel/framework": "~6.0|~7.0|~8.0"
+        "laravel/framework": "~6.0|~7.0|~8.0|^9.0"
```
Comment (omalizadeh) [inline composer.json:26]: wouldn't be better to use caret ^ for all versions? more consistent & cleaner i suppose.

**Pull request merged**